core/rpc/src/author/mod.rs

```diff
@@ -72,6 +74,10 @@ pub trait AuthorApi<Hash, BlockHash> {
 	/// Unsubscribe from extrinsic watching.
 	#[pubsub(subscription = "author_extrinsicUpdate", unsubscribe, name = "author_unwatchExtrinsic")]
 	fn unwatch_extrinsic(&self, metadata: Option<Self::Metadata>, id: SubscriptionId) -> Result<bool>;
+
+	/// Add session key.
```

Comment:
Does it add a key or `set` it? Setting implies the previous one is being replaced, might be worth to rename the method?